pyracing/helpers.py

```diff
@@ -7,7 +7,10 @@
 
 # iRacing has all of their timestamps in ms so we need to divide
 def datetime_from_iracing_timestamp(timestamp):
-    return datetime.utcfromtimestamp(int(timestamp) / 1000)
+    if type(timestamp) is int or type(timestamp) is float:
```

Comment:
The reason I am parsing to `int` here is that since we are getting it from an API, it could come back as a string. Rather than restrict the types it can be parsed from, I would rather just `Except` the errors that we might expect to happen and return a none in those cases.

Reply:
```suggestion
    try:
```
Here we try instead of check if it is a number.